packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -642,56 +713,199 @@ abstract class TableBase extends Resource implements ITable {
+    }
+
+    const usingMetrics: Record<string, cloudwatch.IMetric> = {};
+
+    for (const operation of operations) {
+
+      // the name must start with a lowercase letter.
+      usingMetrics[operation.toLowerCase()] = this.metric('SystemErrors', {
+        statistic: 'sum',
+        dimensions: {
+          TableName: this.tableName,
+          Operation: operation,
+        },
+        ...props,
+      });
+    }
+
+    const sum = new cloudwatch.MathExpression({
+      expression: `${Object.keys(usingMetrics).join(' + ')}`,
+      usingMetrics,
+      color: props?.color,
+      label: props?.label ?? 'Sum over all Operations',
+      period: props?.period,
+    });
+
+    return sum;
   }
 
   /**
-   * Metric for the user errors this table
-   *
-   * @default sum over a minute
+   * Metric for the user errors. Note that this metric reports user errors across all
+   * the tables in the account and region the table resides in.
    */
   public metricUserErrors(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
-    return this.metric('UserErrors', { statistic: 'sum', ...props });
+
+    if (props?.dimensions) {
+      throw new Error('"dimensions is not supported for the "UserErrors" metric');
+    }
+
+    // overriding 'dimensions' here because this metric is an account metric.
+    // see 'UserErrors' in https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/metrics-dimensions.html
+    return this.metric('UserErrors', { statistic: 'sum', dimensions: {}, ...props });
   }
 
   /**
    * Metric for the conditional check failed requests this table
-   *
-   * @default sum over a minute
    */
   public metricConditionalCheckFailedRequests(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
     return this.metric('ConditionalCheckFailedRequests', { statistic: 'sum', ...props });
   }
 
   /**
-   * Metric for the successful request latency this table
-   *
-   * @default avg over a minute
+   * Metric for the successful request latency for specific operations of this table
+   */
+  public metricSuccessfulRequestLatency(props: OperationalMetricOptions): cloudwatch.IMetric {
+
+    if (props?.dimensions?.Operation && props.operations) {
+      // these two represent the same thing, opted to require just one of them instead of dealing with merges
+      throw new Error("Operation can only be specified using the 'operations' property, not the Operation dimension.");
+    }
+
+    const operations = [];
+
+    if (props.dimensions?.Operation) {
+      // user specified the operation dimension, use it.
+      // note this means that props.operations is empty because of the above validation.
+      operations.push(props.dimensions?.Operation);
+    }
+
+    if (props.operations) {
+      // user specified the operations property, use it.
+      // note this means that the Operation dimension is not passed because of the above validation.
+      operations.push(...props.operations);
+    }
```

Comment:
I'd probably reduce this down quite a bit (in both places); the either/or-ness of them has already been explained just above.